Loading a URDF fail if `LD_DEBUG=libs` is set in the env variable

If I set `LD_DEBUG=libs` is set, the loading of URDF file is failing with error:

~~~
ImportError while loading conftest '/home/straversaro-iit.local/jaxsim/tests/conftest.py'.
tests/conftest.py:6: in <module>
    import rod
.pixi/envs/tasks-gpu/lib/python3.12/site-packages/rod/__init__.py:136: in <module>
    check_compatible_sdformat(specification_version="1.10")
.pixi/envs/tasks-gpu/lib/python3.12/site-packages/rod/__init__.py:123: in check_compatible_sdformat
    xmltodict.parse(
.pixi/envs/tasks-gpu/lib/python3.12/site-packages/xmltodict.py:359: in parse
    parser.Parse(xml_input, True)
E   xml.parsers.expat.ExpatError: junk after document element: line 2, column 3
~~~

This is a problem of using an external program instead of a proper library (see https://github.com/ami-iit/rod/issues/30). It may be possible to mitigate/solve the problem by only considering the stdout output instead of the stderr, but I did not checked.

--- a/src/rod/utils/gazebo.py
+++ b/src/rod/utils/gazebo.py
@@ -106,8 +106,7 @@ class GazeboHelper:
                 cp = subprocess.run(
                     [str(gazebo_executable), "sdf", "-p", fp.name],
                     text=True,
-                    stdout=subprocess.PIPE,
-                    stderr=subprocess.STDOUT,
+                    capture_output=True,
                     check=True,
                 )
             except subprocess.CalledProcessError as e:

